apps/price_pusher/docker-compose.beta.sample.yaml

```diff
@@ -1,9 +1,10 @@
 services:
   price-pusher:
-    # Use this to build the price pusher from source. A dependency of the pusher docker
-    # image is the pyth-crosschain monorepo lerna docker image. Build lerna image
-    # using the following command from the repo root:
-    # `docker buildx build -f Dockerfile.lerna -t lerna .`
+    # Use this to build the price pusher from source. The pusher docker image
```

Comment:
I think we should update these.

Reply:
Updated @ali-bahjati , see https://github.com/pyth-network/pyth-crosschain/pull/1686/commits/fe0e96f4f53f21d5518e282719e96515d4eeea5a